frontend/src/pages/ActivityReport/Pages/Review/ReviewItem.js

```diff
@@ -40,17 +42,37 @@ export const mapUrlValue = (v) => {
   return result;
 };
 
+const onHubFileLinkClick = (e) => {
+  e.preventDefault();
+  window.open(e.target.href, '_blank');
+};
+
 const ReviewItem = ({
-  label, name, path, sortValues,
+  label, name, path, sortValues, customValue, linkNamePath, isFile, isRichText,
 }) => {
   const { watch } = useFormContext();
-  const value = watch(name);
-  let values = value;
+  let value = null;
+  let values = [];
+  let linkNameValues = [];
+  if (!customValue) {
+    value = watch(name);
+    values = value;
+  } else {
+    value = customValue[name];
+    values = value;
+  }
```

Comment:
Can you explain what is going on here, via a comment?

Reply:
done